client/src/components/App.jsx

```diff
@@ -14,6 +14,9 @@ export default function App() {
   const [selectedProduct, setSelectedProduct] = useState(products[0]); // For selecting the current product to be shown
 
   useEffect(() => {
+    document.body.addEventListener('click', e => {
+      console.log('EVENT: ', e);
+    });
```

Comment:
Will be our eventual click handler? This should work at every (after we connect it to state) level right?